pkg/sdk/testint/schemas_integration_test.go

```diff
@@ -156,25 +281,100 @@ func TestInt_SchemasAlter(t *testing.T) {
+		assertParameterEquals(t, params, sdk.AccountParameterMaxDataExtensionTimeInDays, "42")
+		assertParameterEquals(t, params, sdk.AccountParameterExternalVolume, externalVolumeTest.Name())
+		assertParameterEquals(t, params, sdk.AccountParameterCatalog, catalogIntegrationTest.Name())
+		assertParameterEquals(t, params, sdk.AccountParameterReplaceInvalidCharacters, "true")
+		assertParameterEquals(t, params, sdk.AccountParameterDefaultDDLCollation, "en_US")
+		assertParameterEquals(t, params, sdk.AccountParameterStorageSerializationPolicy, string(sdk.StorageSerializationPolicyCompatible))
+		assertParameterEquals(t, params, sdk.AccountParameterLogLevel, string(sdk.LogLevelInfo))
+		assertParameterEquals(t, params, sdk.AccountParameterTraceLevel, string(sdk.TraceLevelOnEvent))
+		assertParameterEquals(t, params, sdk.AccountParameterSuspendTaskAfterNumFailures, "10")
+		assertParameterEquals(t, params, sdk.AccountParameterTaskAutoRetryAttempts, "10")
+		assertParameterEquals(t, params, sdk.AccountParameterUserTaskManagedInitialWarehouseSize, string(sdk.WarehouseSizeMedium))
+		assertParameterEquals(t, params, sdk.AccountParameterUserTaskTimeoutMs, "12000")
+		assertParameterEquals(t, params, sdk.AccountParameterUserTaskMinimumTriggerIntervalInSeconds, "30")
+		assertParameterEquals(t, params, sdk.AccountParameterQuotedIdentifiersIgnoreCase, "true")
+		assertParameterEquals(t, params, sdk.AccountParameterEnableConsoleOutput, "true")
+		assertParameterEquals(t, params, sdk.AccountParameterPipeExecutionPaused, "true")
+
+		err = client.Schemas.Alter(ctx, schemaTest.ID(), &sdk.AlterSchemaOptions{
+			Unset: &sdk.SchemaUnset{
+				DataRetentionTimeInDays:                 sdk.Bool(true),
+				MaxDataExtensionTimeInDays:              sdk.Bool(true),
+				ExternalVolume:                          sdk.Bool(true),
+				Catalog:                                 sdk.Bool(true),
+				ReplaceInvalidCharacters:                sdk.Bool(true),
+				DefaultDDLCollation:                     sdk.Bool(true),
+				StorageSerializationPolicy:              sdk.Bool(true),
+				LogLevel:                                sdk.Bool(true),
+				TraceLevel:                              sdk.Bool(true),
+				SuspendTaskAfterNumFailures:             sdk.Bool(true),
+				TaskAutoRetryAttempts:                   sdk.Bool(true),
+				UserTaskManagedInitialWarehouseSize:     sdk.Bool(true),
+				UserTaskTimeoutMs:                       sdk.Bool(true),
+				UserTaskMinimumTriggerIntervalInSeconds: sdk.Bool(true),
+				QuotedIdentifiersIgnoreCase:             sdk.Bool(true),
+				EnableConsoleOutput:                     sdk.Bool(true),
+				PipeExecutionPaused:                     sdk.Bool(true),
+			},
+		})
 		require.NoError(t, err)
-		assert.Equal(t, comment, *s.Comment)
+
+		params = testClientHelper().Parameter.ShowSchemaParameters(t, schemaTest.ID())
+		assertParameterEqualsToDefaultValue(t, params, sdk.ObjectParameterDataRetentionTimeInDays)
```

Comment:
please add this also to the create minimal test

Reply:
Done.